lib/client/index.js

```diff
@@ -0,0 +1,1710 @@
+'use strict';
+
+var $ = (global && global.$) || require('jquery');
+var _ = require('lodash');
+var d3 = (global && global.d3) || require('d3');
+
+var language = require('../language')();
+var profile = require('../profilefunctions')();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['profile' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/799298422/issues/source?bid=2452379&fileBranchId=2725007#l8)